cfme/tests/infrastructure/test_individual_host_creds.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+import cfme.fixtures.pytest_selenium as sel
+import cfme.web_ui.flash as flash
+import pytest
+import random
+import utils.conf as conf
+
+from cfme.infrastructure.host import credential_form
+from cfme.infrastructure.provider import details_page
+from cfme.web_ui import fill
+from utils import testgen
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = testgen.provider_by_type(metafunc, ['virtualcenter', 'rhevm'])
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist, scope="module")
+
+
+# Tests to automate BZ 1278904
+def test_single_host_bad_creds(setup_provider, provider):
+    """  Tests multiple host credentialing  with good credentials """
+
+    sel.force_navigate('infrastructure_provider', context={'provider': provider})
+    sel.click(details_page.infoblock.element("Relationships", "Hosts"))
```

Comment:
Why do we navigate into the Hosts page for the provider here? line 28 should be where this happens if the "filtering" by provider needs to happen at all. Hopefully the hostnames will be unique and so there would be no need to do these separate navigations.